### ISSUE TITLE
Visualization examples to explore / consider

### ISSUE THREAD

**Issue by jmcmurry** [monarch-initiative/monarch-legacy#1176]
Not sure if there's anything we can learn from this visualization, but it is absolutely spectacular. Just filing it here for future exploration.

http://mbtaviz.github.io/

**Comment by harryhoch** [comment]
yes. very cool. the schedule graph at the top is clearly inspired by Marey’s Paris train schedule, as described by Ed Tufte (http://www.edwardtufte.com/bboard/q-and-a-fetch-msg?msg_id=0003zP), but it’s the coordination of the various visualizations and the use of the shading that really shine here. Definitely some ideas to learn from.

Credits indicate that this was done in Matt Ward’s vis class @ WPI. He’s been doing nice work in this area for years.

**Comment by jmcmurry** [comment]
More examples of nicely done synchronized views that are rich but sensible.
http://casestudies.brain-map.org/celltax#section_explorea
